### PR TITLE
[release-1.0] e2e, hotplug ifaces: Assert absent ifaces removed from spec of stopped VMs

### PR DESCRIPTION
**Pull request by ormergi**
**What this PR does / why we need it**:

Interfaces set with state 'absent' of stopped VMs are expected to be removed from spec.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/network/hotplug.go [hunk 1]
```diff
@@ -434,20 +434,21 @@ var _ = SIGDescribe("nic-hotunplug", func() {
...
 				Expect(err).NotTo(HaveOccurred())
+
 				iface := vmispec.LookupInterfaceByName(vm.Spec.Template.Spec.Domain.Devices.Interfaces, linuxBridgeNetworkName2)
-				return iface.State
-			}, 30*time.Second).Should(Equal(v1.InterfaceStateAbsent))
+				g.Expect(iface).To(BeNil(), "absent iface should be removed from stopped VM spec")
+			}, 30*time.Second).Should(Succeed())
 
-			Expect(previousVMTemplateSpec.Networks).To(Equal(vm.Spec.Template.Spec.Networks), "network spec should not change")
+			iface := libnet.LookupNetworkByName(vm.Spec.Template.Spec.Networks, linuxBridgeNetworkName2)
```
Comment (EdDev) [inline tests/network/hotplug.go:450]: I do not think we remove the network, otherwise the flake would have been constantly failing, not like now.

Comment (ormergi) [reply tests/network/hotplug.go:450]: Networks are expected to be removed as well https://github.com/kubevirt/kubevirt/blob/release-1.0/pkg/virt-controller/watch/vm.go#L2663-L2665.
I think now it doesn't even reach the spec networks assert.

Comment (EdDev) [reply tests/network/hotplug.go:450]: I think I now understand why it worked before and it is flaky.
The flake was due to the fact that we catched the interface before it was removed and therefore the network was still there.

So these two come as pairs.

**Comment by EdDev** [comment]
Why is it a draft?

/lgtm

**Pull request merged**